packages/docs-app/src/examples/select-examples/multiSelectExample.tsx

```diff
@@ -139,6 +139,8 @@ export class MultiSelectExample extends React.PureComponent<ExampleProps, MultiS
                     onItemSelect={this.handleFilmSelect}
                     onItemsPaste={this.handleFilmsPaste}
                     popoverProps={{ matchTargetWidth, minimal: popoverMinimal }}
+                    // Due to usePortal, can't neatly use className from _examples.scss
+                    popoverContentProps={this.state.customTarget ? { style: { maxWidth: 400 } } : undefined}
```

Comment:
Need to explicitly define a maxWidth when using `usePortal` + `TagInput` is inside the Popover.

There is no width defined in any of TagInput's grandparents when it's rendered through usePortal so it will never flex-wrap, and infinitely grow horizontally